Tests.Integration.Presentation.Web/Contract/V2/ItContractsApiV2Test.cs

```diff
@@ -2155,6 +2158,7 @@ private async Task<(IdentityNamePairResponseDTO contractType, IdentityNamePairRe
             Assert.Equal(request?.Validity?.ValidTo?.Date, freshDTO.General.Validity?.ValidTo);
             Assert.Equal(request?.Validity?.ValidFrom?.Date, freshDTO.General.Validity?.ValidFrom);
             Assert.Equal(request?.Validity?.EnforcedValid == true, freshDTO.General.Validity?.EnforcedValid == true);
+            Assert.Equal(request?.ProcurementInitiated, freshDTO.General.ProcurementInitiated);
```

Comment:
Once the data has been written once, the state in the backend will be undecided since null is mapped to undecided in the write mapper.. that's fine, but your test will fail with this way of doing it.. try looking at the other tests